versioned_docs/version-3.0.x/security-openid-connect.md

```diff
@@ -68,6 +72,10 @@ When using OIDC for a client connecting through the proxy to the broker, it is n
 
 :::
 
+:::note
+The Pulsar WebSocket Proxy does not yet support OpenID Connect authentication. Here is an issue tracking this feature: [#20236](https://github.com/apache/pulsar/issues/20236).
```

Comment:
This note wasn't added to the next version of docs. Does it imply it's going to be supported soon?

Reply:
Yes, I submitted the fix here https://github.com/apache/pulsar/pull/20238. It's not really a bug though, so it'll ship in Pulsar 3.1.0.